src/Sulu/Bundle/ContactBundle/Tests/Unit/CustomerIdConverterTest.php

```diff
@@ -56,7 +56,7 @@ public function convertGroupedIdsToIdsProvider()
     /**
      * @dataProvider convertGroupedIdsToIdsProvider
      */
-    public function testConvertGroupedIdsToIds($groupedIds, $expected): void
+    public function testConvertGroupedIdsToIds(array $groupedIds, array $expected): void
```

Comment:
This is not a bc break because the function where those variables are used already have a return type of array. So the only thing is we fail the code faster on broken types.